ETC.tsv

```diff
@@ -162,7 +162,7 @@ ETC_20150317_000161	Wheelen
 ETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
 ETC_20150317_000163	Wheelen 3
 ETC_20150317_000164	Mauros
-ETC_20150317_000165	The stone calvalry of the Royal Mausoleum are magically assembled to withstand even the heavily armed knight.
+XETC_20150317_000165	The stone ETC_20150317_000165	 of the Royal Mausoleum are magically assembled to withstand even the heavily armed knight.
```

Comment:
Check this again, you added the code to the file into the sentence for some reason